libr/anal/esil.c

```diff
@@ -2641,112 +2635,6 @@ static int signed_compare_gt(ut64 a, ut64 b, ut64 size) {
 	return result;
 }
 
-static int esil_smaller(RAnalEsil *esil) { // 'dst < src' => 'src,dst,<'
```

Comment:
dont remove them yet. use #if 1 and #warning deprecate, first go for ==, im not convinced about the <= yet because its more readable